src/service/cid-api.js

```diff
@@ -141,7 +143,7 @@ export class GoogleCidApi {
       setCookie(this.win_, cookieName, res['clientId'], this.expiresIn_(YEAR));
       return res['clientId'];
     } else {
-      this.persistToken_(TokenStatus.ERROR, DAY);
+      this.persistToken_(TokenStatus.NOT_FOUND, DAY);
```

Comment:
did we? I thought you wanted to see the QPS with 1 day first.
But I'm totally supportive to reduce this time.